documentation/dotnet-trace-instructions.md

```diff
@@ -0,0 +1,131 @@
+# Trace for performance analysis utility (dotnet-trace)
+
+The dotnet-trace tool is a cross-platform CLI global tool that enables the collection of .NET Core traces of a running process without any native profiler involved. It is built around the EventPipe technology of the .NET Core runtime as an alternative to ETW on Windows and LTTNG on non-Windows.
+
+## Installing dotnet-trace
+
+The first step is to install the dotnet-trace CLI global tool. This requires at least version 2.1 of the SDK to be installed.
+
+```cmd
+$ dotnet tool install --global dotnet-trace --version 1.0.3-preview5.19217.3 --add-source https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json
+You can invoke the tool using the following command: dotnet-trace
+Tool 'dotnet-trace' (version '1.0.3-preview5.19217.3') was successfully installed.
+```
+
+## Using dotnet-trace
+
+In order to collect traces you will need to:
+
+- First, find out the process identifier (pid) of the .NET Core 3.0 app to collect traces from
+
+  - On Windows, there are options such as using the task manager or the `tasklist` command on the cmd window.
+  - On Linux, the trivial option could be using `pidof` on the terminal window.
+
+- Then, run the following command:
+
+```cmd
+dotnet-trace collect --process-id <PID> --providers Microsoft-Windows-DotNETRuntime
+
+Press <Enter> to exit...
+Connecting to process: <Full-Path-To-Process-Being-Profiled>/dotnet.exe
+Collecting to file: <Full-Path-To-Trace>/trace.netperf
+  Session Id: <SessionId>
+  Recording trace 721.025 (KB)
+```
+
+- Finally, stop collection by pressing the \<Enter> key, and *dotnet-trace* will finish logging events to *trace.netperf* file.
+
+## Commonly used keywords for the *Microsoft-Windows-DotNETRuntime* provider
+
+ Runtime keyword name           | Keyword Value     | Description
+ ------------------------------ | ----------------: | ------------
+None                            |                 0 |
+All                             |  FFFFFFFFFFFFFFBF | All does not include start-enumeration.  It just is not that useful.
```

Comment:
It is just not that useful.

Reply:
I want to trim this table. Any suggestions?

Reply:
Trim as in reformat or as in remove rows? I think it would be OK if you wanted to remove the table entirely or if you wanted to provide only a few examples (perhaps GC, Jit, and threadpool).